### PR TITLE
Initial tasks for Outreachy

### PR DESCRIPTION
**Pull request by navyagarwal**
Completed tasks #2, #3 , #4 , #5

### CODE REVIEW

File: 2023-round-1/navyagarwal/nx_version.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+2.8.8
```
Comment (paulitapb) [inline 2023-round-1/navyagarwal/nx_version.txt:1]: Could you upgrade to 3.0?

Comment (navyagarwal) [reply 2023-round-1/navyagarwal/nx_version.txt]: Yeah, done!

File: 2023-round-1/navyagarwal/nx_tutorial_script.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+G.add_nodes_from(['a', 'b'])       # from list
+G.add_nodes_from(range(9, 11))     # in range
+
+X = nx.path_graph(2)
+Y = nx.complete_graph(3)
+
+G.add_nodes_from(X)                # from graph X
+G.add_node(Y)                      # graph Y
+
+
```
Comment (MridulS) [inline 2023-round-1/navyagarwal/nx_tutorial_script.py:21]: Any specific reason for doing this?

Wouldn't this just end up adding the node 0, 1, 2 again (which won't really make any difference to the Graph `G`)

File: 2023-round-1/navyagarwal/nx_tutorial_script.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+nx.draw(G, with_labels = True, node_color = 'yellow')
+
+
+# printing shortest path between all pairs of nodes if it exists
+for source in G:
+    for dest in G:
+        if(nx.has_path(G, source, dest)):
+            print(source, " -> ", dest," : ", nx.shortest_path(G, source, dest))
```
Comment (MridulS) [inline 2023-round-1/navyagarwal/nx_tutorial_script.py:49]: You can also use `all_pairs_shortest_path` to compute the paths in one go and then iterate over the dictionary to print our the results.

There is nothing wrong with this approach too! Just wanted to let you know there exists another function.

Comment (navyagarwal) [reply 2023-round-1/navyagarwal/nx_tutorial_script.py:49]: Yeah, just referred the doc again, I hadn't checked out the advanced interface algorithms for shortest paths, thanks! :)

**Comment by MridulS** [comment]
Thanks @navyagarwal!

**Pull request merged**